addons/markers/functions/fnc_placeMarker.sqf

```diff
@@ -16,15 +16,27 @@
  * Public: No
  */
 
-disableserialization;
 params ["_display", "_closeNum"];
 TRACE_2("params",_display,_closeNum);
 
 if (_closeNum isEqualTo 1) then {
     if !(GVAR(editingMarker) isEqualTo "") then {
         //delete "old" marker
         deleteMarker GVAR(editingMarker);
+    } else {
+        [{
+            // provide hook for external scripts
+            private _newestMarker = allMapMarkers select (count allMapMarkers - 1);
```

Comment:
Are we sure that only one marker is created? This seems very error prone.

Reply:
No, in fact my first thought was that this might very well return an ACE_BFT_x marker since they are updated so often. On the other hand I have never had any issues while moving markers, which works the same way:

https://github.com/acemod/ACE3/blob/master/addons/markers/functions/fnc_placeMarker.sqf#L31